server/cluster/cluster.go

```diff
@@ -273,11 +273,55 @@ func (c *RaftCluster) Start(s Server) error {
 	go c.syncRegions()
 	go c.runReplicationMode()
 	go c.runMinResolvedTSJob()
+	go c.runSyncConfig()
 	c.running = true
 
 	return nil
 }
 
+// runSyncConfig runs the job to sync config.
+func (c *RaftCluster) runSyncConfig() {
+	defer logutil.LogPanic()
+	defer c.wg.Done()
+
+	ticker := time.NewTicker(time.Minute)
+	defer ticker.Stop()
+	stores := c.GetStores()
+
+	index := syncConfig(c.storeConfigManager, stores, 0)
+	for {
+		select {
+		case <-c.ctx.Done():
+			log.Info("sync store config job is stopped")
+			return
+		case <-ticker.C:
+			index = syncConfig(c.storeConfigManager, stores, index)
+			if index >= len(stores) {
+				index = 0
+				stores = c.GetStores()
+			}
+		}
+	}
+}
+
+func syncConfig(manager *config.StoreConfigManager, stores []*core.StoreInfo, index int) int {
```

Comment:
What is the difference between looping all stores and always picking the first one?

Reply:
I think there is no difference between them, the configuration about region size must be some in all tikv store. 
It will be chaos if anyone tikv has different configuration about region size.

Reply:
Then, why not always start from the beginning?